### ISSUE TITLE
Fix jsx-a11y/no-static-element-interactions warnings raised by jsx-a11y linter.

### ISSUE THREAD

**Issue by yzen** [firefox-devtools/debugger#7537]
To get a list of accessibility warnings raised by jsx-a11y linter run: `yarn lint:jsx-a11y`.

From that list please note all instances that are flagged to have "jsx-a11y/no-static-element-interactions" problem. Each one needs to be addressed individually.

Once all of the instances are resolved make sure to change this setting in `.eslintrc.jsx-a11y` from:
```
"jsx-a11y/no-static-element-interactions": ["warn", {
  "handlers": ["onClick", "onMouseDown", "onMouseUp", "onKeyPress", "onKeyDown", "onKeyUp"]
}],
```

to
```
"jsx-a11y/no-static-element-interactions": ["error", {
  "handlers": ["onClick", "onMouseDown", "onMouseUp", "onKeyPress", "onKeyDown", "onKeyUp"]
}],
```

**Comment by Garbee** [comment]
So [this error](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md) is essentially, not using buttons for actions and instead using divs/spans/etc.

What we want to do here as detailed is go through the warnings, assess the context of the markup, and devise a strategy to handle the cases as consistently as possible. For example, if we have something making an image or a span with an onClick handler, we'd change those to buttons with the text/image node inside of that.

**Comment by yzen** [comment]
That's correct. Buttons is just one of the interactive elements, there's also anchor and other types of inputs (radios, checkboxes) etc. In worst case scenario, and highly discouraged, ARIA role can be used to set the semantics of the interactive element.

**Comment by Garbee** [comment]
Yup, we need to chose the appropriate interactive element to used depending on the task. But, we'd be better off taking advantage of them rather than using aria roles and scripting all the extra needs manually.

**Comment by phoenixking25** [comment]
@ash-tamraz @Garbee is this issue free ?

**Comment by ash-tamraz** [comment]
@phoenixking25 No, I'm still working on it and close to having a patch ready.

**Comment by steffiau** [comment]
@yzen @ash-tamraz I'd like to help on this if it isn't being worked on anymore?

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Also, supporting JSX in the code base is not on our long term roadmap, so this might be invalid report.

Honza
